Examples/test-suite/overload_simple.i

```diff
@@ -166,10 +166,17 @@ char *fid(double, double) {
 %}
 
 %inline %{
-unsigned long long ull() { return 0ULL; }
-unsigned long long ull(unsigned long long ull) { return ull; }
-long long ll() { return 0LL; }
-long long ll(long long ull) { return ull; }
+int sizeof_long() { return sizeof(long); }
+
+unsigned long as_ul() { return 0UL; }
+unsigned long as_ul(unsigned long ul) { return ul; }
+long as_l() { return 0L; }
+long as_l(long l) { return l; }
```

Comment:
Just `l` was not allowed because of https://www.flake8rules.com/rules/E743.html , so I've renamed it to as_l.

Reply:
Hard to tell from the diffs displayed here, but `ull()` is now `as_ull()` and `ull()` is now `as_ll()`.